project/settings.py

```
@@ -37,11 +42,44 @@
+# Domain for Jitsi video conferencing
+JITSI_DOMAIN = config('JITSI_DOMAIN', default='meet.jit.si')
+
+# Prefix for Jitsi room names
+JITSI_ROOM_PREFIX = config('JITSI_ROOM_PREFIX', default='myapp-')
+
+# Magic cookie for Jitsi authentication
+JITSI_MAGIC_COOKIE = config('JITSI_MAGIC_COOKIE', default=None)
+
+# Raise an error if the magic cookie is not set
+if JITSI_MAGIC_COOKIE is None:
+    raise ValueError("JITSI_MAGIC_COOKIE environment variable is required for video conferencing")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)